CHANGES.rst

```diff
@@ -1,3 +1,517 @@
+Version 6.0.0 (2023-11-25)
+==========================
+
+New Features
+------------
+
+astropy.config
+^^^^^^^^^^^^^^
+
+- The new ``ConfigNamespace.help()`` method provides a convenient way to get
+  information about configuration items. [#13499]
+
+astropy.coordinates
+^^^^^^^^^^^^^^^^^^^
+
+- Support has been added to create geodetic representations not just for existing ellipsoids
+  from ERFA, but also with explicitly provided values, by defining a subclass of
+  ``BaseGeodeticRepresentation`` with the equatorial radius and flattening assigned to
+  ``_equatorial_radius`` and ``_flattening`` attributes. [#14763]
+
+- Add ``BaseBodycentricRepresentation``, a new spheroidal representation for bodycentric
+  latitudes and longitudes. [#14851]
+
+- Support Numpy broadcasting over frame data and attributes. [#15121]
+
+astropy.cosmology
+^^^^^^^^^^^^^^^^^
+
+- Added a ``write_latex()`` method for exporting a Cosmology object to a LaTex table. [#14701]
+
+- Added argument ``rename`` to Cosmology's I/O, allowing for input and output symbols to
+  be renamed. [#14780]
+
+- All non-abstract Cosmology subclasses are now automatically registered to work with
+  Astropy's YAML serialization. [#14979]
+
+- Cosmology I/O now auto-identifies the '.tex' suffix with the 'ascii.latex' format. [#15088]
+
+- The ``Cosmology`` class now has a new property to access the parameters of the
+  cosmology: ``.parameters``. This property return a read-only dictionary of all the
+  non-derived parameter values on the cosmology object. When accessed from the class (not
+  an instance) the dictionary contains ``Parameter`` instances, not the values. [#15168]
+
+- The field ``default`` has been added to ``Parameter``. This can be used to introspect
+  the default value of a parameter on a cosmology class ``e.g. LambdaCDM.H0.default``. [#15400]
```

Comment:
```suggestion
  the default value of a parameter on a cosmology class e.g. ``LambdaCDM.H0.default``. [#15400]
```

Reply:
Why weren't these caught in the original PR?